data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,59 @@ mission "Arachi Cartoonist 2"
+			`	The central figure is a young Saryd boy, galloping through a dense forest. Sound effects of thunder begin and grow increasingly frequent, echoing around you as if in a real thunderstorm. A white piece of paper, shaped like a lightning bolt, comes down swiftly and strikes one of the trees in the boy's way. Flame-shaped folds rain down from the top of the stage and stick to the tree as it falls, blocking the boy's path. He turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore. Hurt by some of the flames, he is now limping through the forest.`
+			`	Whimpering as the thunder grows closer once more, he spots a tree unlike the others, a great oak with branches thicker than the trees around him. He goes to it and lays down at its base, and soon he falls asleep as the curtain closes.`
+			`	It opens once again after a few moments, with the boy waking up in darkness. When he gets up, rumbling sounds are accompanied by rays of light reaching the ground, and he realizes the great oak had lowered its branches around him. As it lifts them up, it knocks back several trees that had fallen on them during the storm. The sun as his guide, the boy manages to leave the forest, and the curtains close again.`
+				goto end
+			label pearl
+			`	Once you have the seashell in your hands, the organizers tell you to go through the path in the middle, and you do so.`
+			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below, or strings from above. There are figures close to where the curtains were, and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`
+			`	A Saryd is on top of some sort of long, wooden platform, with a deep blue background under it. Only when you see the fishing rod do you realize that he is on a boat, fishing at sea. He feels a bite, but when he reels it in there is nothing on the hook. He feels another bite, and again finds nothing. Time and again, he looks at the hook to find the bait still there, intact, yet he continues to feel bites that are sometimes so blatant even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`
```

Comment:
```suggestion
			`	A Saryd is on top of some sort of long, wooden platform, with a deep blue background under it. Only when you see the fishing rod do you realize that he is on a boat, fishing at sea. He feels a bite, but when he reels it in, there is nothing on the hook. He feels another bite and again finds nothing. Time and again, he looks at the hook to find the bait still there, intact, yet he continues to feel bites that are sometimes so noticeable even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`
```
I feel like `noticeable` or another word may be a little more appropriate for what you're trying to say rather than `blatant`. Just my take though